CMakeLists.txt

```diff
@@ -15,6 +15,9 @@ set(CMAKE_INSTALL_PREFIX "${CMAKE_CURRENT_SOURCE_DIR}/install" CACHE PATH "CMake
 
 cmake_policy(SET CMP0048 NEW)
 cmake_minimum_required(VERSION 3.12.0)
+# note: version should not be changed manually, use bump-my-version instead:
+# install: pip install bump-my-version
+# example: bump-my-version bump patch
```

Comment:
Just a suggestion in case you don't know, it's not really necessary to install `bump-my-version`, using `pipx` or `uv` is trivial to run it directly from PyPI:
```python
# with pipx
pipx run bump-my-version patch
# with uv
uvx run bump-my-version patch
```

Reply:
thanks, didn't know. here I left it as it's more common to know pip than pipx I guess, but I know you like to use these places to educate people. ;)